docs/tutorial/update.rst

```diff
@@ -0,0 +1,64 @@
+Update management
+~~~~~~~~~~~~~~~~~
+
+Microceph will not manages its own updates - users are required to do so. In order to set up microceph to manage updates correctly, the following steps need to be taken.
```

Comment:
I feel like users who are not familiar with snaps would benefit from some context around snap updates. Also, I don't think we should make it an imperative -- a single node MicroCeph cluster should be fine with auto-upgrades (if the upgrade works at all). Maybe something like:

----

MicroCeph updates are managed by the snap mechanisms. By default snaps will automatically deploy updates, this however can be problematic for MicroCeph clusters due to ordering of and concurrent rollout of updates. It's therefore strongly advised to disable automatic updates for MicroCeph snaps. For details on the snap update mechanism look [here](https://snapcraft.io/docs/keeping-snaps-up-to-date)

----